tests/wsl/wsl_cmd_check.pm

```diff
@@ -34,6 +34,10 @@ sub run {
         $self->run_in_powershell(cmd => 'wsl -u root zypper -q -n in python3', timeout => 120);
         $self->run_in_powershell(cmd => q{wsl python3 -c "print('Hello from Python living in WSL')"});
     }
+    # Check if wsl_gui has been installed during the YaST2 firstboot
+    $self->run_in_powershell(cmd => 'wsl /bin/bash -c "zypper se -t pattern wsl_gui | grep -i i+"') if (get_var('WSL_GUI'));
+    # Check if SLED has been enabled during the YaST2 firstboot
+    $self->run_in_powershell(cmd => 'wsl /bin/bash -c "cat /etc/os-release | grep -i desktop"') if (check_var('SLE_PRODUCT', 'sled'));
```

Comment:
Here is a brief check for both new features.

Reply:
```
❯ zypper -q se -i -t pattern base1; echo $?
No matching items found.
104
❯ zypper -q se -i -t pattern base; echo $?

S  | Name          | Summary                | Type
---+---------------+------------------------+--------
i+ | base          | Base System            | pattern
i+ | enhanced_base | Enhanced Base System   | pattern
i+ | minimal_base  | Minimal Appliance Base | pattern
0
```